TestCases/compliance-level-3/0082-feel-coercion/0082-feel-coercion.dmn

```diff
@@ -0,0 +1,505 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0082-feel-coercion" name="0082-feel-coercion" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <description>FEEL type conformance of DT and BKM results</description>
+
+    <itemDefinition name="tNumberList" isCollection="true">
+        <typeRef>number</typeRef>
+    </itemDefinition>
+
+    <itemDefinition name="tStringList" isCollection="true">
+        <typeRef>string</typeRef>
+    </itemDefinition>
+
+    <itemDefinition name="tDS_001">
+        <itemComponent name="decision_ds_001">
+            <typeRef>string</typeRef>
+        </itemComponent>
+    </itemDefinition>
+
+    <itemDefinition name="tNameAndAge">
+        <itemComponent name="name">
+            <typeRef>string</typeRef>
+        </itemComponent>
+        <itemComponent name="age">
+            <typeRef>number</typeRef>
+        </itemComponent>
+    </itemDefinition>
+
+    <decision name="decision_001" id="_decision_001">
+        <variable typeRef="string" name="decision_001"/>
+        <literalExpression>
+            <text>1+1</text>
+        </literalExpression>
+    </decision>
+
+<!--    <decision name="decision_002" id="_decision_002">
+        <variable typeRef="list" name="decision_002"/>
+        <literalExpression>
+            <text>[1,2,"foo"]</text>
+        </literalExpression>
+    </decision>-->
+
+    <decision name="decision_003" id="_decision_003">
+        <variable typeRef="tNumberList" name="decision_003"/>
+        <literalExpression>
+            <text>[1,2,"foo"]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_004" id="_decision_004">
+        <variable typeRef="tNameAndAge" name="decision_004"/>
+        <literalExpression>
+            <text>{name: "foo", surname: "bar", age: 10}</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_005" id="_decision_005">
+        <variable typeRef="tNameAndAge" name="decision_005"/>
+        <literalExpression>
+            <text>{name: "foo"}</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_006" id="_decision_006">
+        <variable typeRef="tStringList" name="decision_006"/>
+        <literalExpression>
+            <text>"foo"</text>
```

Comment:
This test seems to me to rely on DMN13-132 hence I wonder if we can truly say it is DMNv1.2